code/modules/mob/dead/observer/observer.dm

```diff
@@ -28,7 +28,7 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 							//If you died in the game and are a ghost - this will remain as null.
 							//Note that this is not a reliable way to determine if admins started as observers, since they change mobs a lot.
 	var/atom/movable/following = null
-	var/fun_verbs = 0
+	var/fun_verbs = 1
```

Comment:
Possession is not the only thing affected by it, as far as I am aware.

Reply:
Fixed it.